tests/test_io/test_pickle/test_pickle.py

```
@@ -94,10 +94,11 @@ class TestScan:
     def test_scan_attrs_eq_read_attrs(self, pickle_patch_path):
         """Ensure read/scan produce the same attrs."""
         scan_list = dc.scan(pickle_patch_path)
-        patch_attrs_list = [x.attrs for x in dc.read(pickle_patch_path)]
+        patch_summaries = [x.summary for x in dc.read(pickle_patch_path)]
 
-        for scan_attrs, patch_attrs in zip(scan_list, patch_attrs_list):
+        for scan_attrs, patch_summary in zip(scan_list, patch_summaries):
```

Ruff (0.15.5)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)